middleware/walletValidator.middleware.js

```diff
@@ -175,5 +181,6 @@ module.exports = {
     paymentRequestValidation: validationMiddleware(schemas.paymentRequestSchema, 'body'),
     getQuoteValidation: validationMiddleware(schemas.getQuoteSchema, 'body'),
     sendTokenValidation: validationCustomMiddleware(schemas.sendTokenSchema, schemas.maxSendTokenQuerySchema),
-    migrationWalletValidation: validationMiddleware(schemas.migrationWalletSchema, 'body')
+    migrationWalletValidation: validationMiddleware(schemas.migrationWalletSchema, 'body'),
+    cryptoListValidation
```

Comment:
I suggest using a validation middleware in the exports object.
cryptoListValidation : validationMiddleware(schemas.cryptoListSchema, 'query');